src/content/manifest/overview-manifest.mdx

```diff
@@ -0,0 +1,153 @@
+
+```yaml
+build:
+  base:
+    context: .
+  api:
+    context: api
+  frontend:
+    context: frontend
+    dockerfile: Dockerfile
+    target: dev
+    depends_on: base
+    args:
+      SOURCE_IMAGE: ${OKTETO_BUILD_BASE_IMAGE}
+    secrets:
+      npmrc: .npmrc
+```
+
+This configuration defines images to be built for three services: `base`, `api`, and `frontend`. It also defines a `context` for each image, which tells Okteto which folder/subfolder to use for building each container image. For example: `default (.)` tells Okteto to use the current folder for the `base` image, the `api` subfolder for the `api` image, and the `frontend` subfolder for the `frontend` image.
+
+You can learn more about these additional parameters [here](reference/okteto-manifest.mdx).
+
+## Deploy
+
+This section tells Okteto how, specifically, to deploy your development environment. It typically uses a combination of `helm`, `kubectl`, and `okteto` commands. You can execute multiple commands and label them.
+
+> Note: If you have external resources you'd like to configure and show within the Okteto development environment, you must configure those in this section.
+
+Your `deploy` section might look like this:
+
+```yaml
+deploy:
+    - name: Deploy PostgreSQL
+    command: helm upgrade --install postgresql postgresql/postgresql-11.6.21.tgz -f postgresql/values.yml --version 11.6.21
+    - name: Deploy Frontend
+    command: helm upgrade --install frontend frontend/chart --set image=${OKTETO_BUILD_FRONTEND_IMAGE}
+    - name: Deploy API
+    command: helm upgrade --install api api/chart --set image=${OKTETO_BUILD_API_IMAGE} --set load=${API_LOAD_DATA:-true}
+```
+
+In this section the manifest instructs Okteto to execute three commands: deploy a postgresql database, deploy the `frontend` image (using an environment variable), and deploy the `api` image (also using environment variables).
+
+These commands are executed within the development environment created by Okteto enabling you to create a production-like environment for developing your application.
+
+## Destroy
+
+The commands you define in this section will ensure any resources external to the Kubernetes cluster Okteto manages are destroyed when deleting the development environment. 
+
+> Note: The Okteto CLI command `okteto destroy` will automatically destroy any Kubernetes resources created by the `okteto deploy` command.
+
+Your `destroy` section might look like this:
+
+```yaml
+destroy:
+  image: okteto/pipeline-runner:1.0.0-sam
+  commands:
+    - name: destroy worker service
+      command: |
+        sam delete --no-prompts --stack-name "${OKTETO_NAMESPACE}-voting-texkhnclxd" --region us-east-1
+```
+
+In this example `destroy` section we are deleting the worker service running on the image container specified in the `image:` attribute. When the development environment is deleted, Okteto will execute this command ensuring orphaned resources don't remain active.
```

Comment:
Should we add a link to the `deploy` reference docs?

Reply:
```suggestion
In this example `destroy` section we are deleting the worker service running on the image container specified in the `image:` attribute. When the development environment is deleted, Okteto will execute this command ensuring orphaned resources don't remain active.

You can learn more about this section [here](reference/okteto-manifest.mdx#deploy-string-optional).
```